oak-query-spi/src/main/java/org/apache/jackrabbit/oak/spi/query/CompositeQueryIndexProvider.java

```diff
@@ -23,8 +23,8 @@
 import org.apache.jackrabbit.oak.spi.state.NodeState;
 import org.jetbrains.annotations.NotNull;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Lists;
+import org.apache.jackrabbit.guava.common.collect.ImmutableList;
```

Comment:
Since we have moved to Java 11, we should use `List.of()` instead of `ImmutableList`.

Reply:
Nope. a) I've decided to do only mechanical changes here. b) There may be cases where we want to avoid unnecessary changes that can not be backported to 1.22. So I'll leave this to the actual "owners" of the module.

Reply:
Nope. a) I've decided to do only mechanical changes here. b) There may be cases where we want to avoid unnecessary changes that can not be backported to 1.22. So I'll leave this to the actual "owners" of the module.